pkg/ts/Runtime.ts

```diff
@@ -8,15 +8,15 @@
 import {Arc} from './core/Arc.js';
 import {Host} from './core/Host.js';
 import {Store} from './core/Store.js';
+import {ArcMeta, ParticleMeta, StoreMeta} from './core/Meta.js';
 import {EventEmitter} from './core/EventEmitter.js';
-import {Paths} from './utils/paths.js';
-import {AggregateStore} from './core/AggregateStore.js';
-import {logFactory} from './utils/log.js';
+import {logFactory, Logger} from './utils/log.js';
```

Comment:
`Logger` to come from `./utils/types.js`

Reply:
I think this would require `logKinds`, `errKinds`, `LoggerFunction` all to be in `types`.

I'll upload that change assuming it is intended, but I'm not sure about it. Keen for your advice & thoughts.